src/main/java/io/jenkins/plugins/conventionalcommits/dto/HelmChart.java

```diff
@@ -0,0 +1,36 @@
+package io.jenkins.plugins.conventionalcommits.dto;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+/**
+ * DTO to represent an helm chart's file.
+ */
+public class HelmChart implements Serializable {
+
+    // Version attribute
+    private String version;
+
+    public String getVersion() {
+        return version;
+    }
+
+    public void setVersion(String version) {
+        this.version = version;
+    }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I usually prefer avoiding one character name variable as it makes it harder to read code.
In this case I would prefer `public boolean equals(Object anotherObject) {` instead of  `public boolean equals(Object o) {`

The only place where I find it acceptable is for iterator such as `for (int i = 0; i < 5; i++) {`